toss_3_x86_64_ib/compilers.yaml

```diff
@@ -38,6 +38,19 @@ compilers::
     modules: []
     environment: {}
     extra_rpaths: []
+- compiler:
+    spec: clang@8.0.1
```

Comment:
I see 8.0.0 below. Is that intentional?

Reply:
I checked most of the specs manually by generating a host-config file with Spack through uberenv. I may have not tested CUDA 12. I'll do that and report back. 

The clang 8.0.1 entry is a mistake. I just pushed a fix. Good catch.

Note that merging this PR is not urgent. We will be updating our CI jobs after the holidays and I wanted to get updates here done first.

Reply:
Also, I'm finishing up a RAJA Dev Guide PR that explains how all the CI stuff works and how to perform basic maintenance tasks. I will tag you as a reviewer on that and would appreciate it if you would look it over for correctness.